django_oikotie/apartment.py

```diff
@@ -0,0 +1,17 @@
+from dataclasses import dataclass
+
+from .housing_company import BaseClass
```

Comment:
There are mostly absolute imports across the project, so I would like to stay uniform, so:

`from django_oikotie.housing_company import BaseClass`

Reply:
👍